packages/suite/src/components/suite/modals/ReduxModal/TransactionReviewModal/TransactionReviewOutputList/TransactionReviewOutput.tsx

```diff
@@ -26,6 +26,8 @@ const getFeeLabel = (networkType: Account['networkType']) => {
             return 'MAX_FEE';
         case 'solana':
             return 'EXPECTED_FEE';
+        case 'stellar':
```

Comment:
move below `ethereum`

Reply:
This issue was addressed during the rebase.